core/src/saros/misc/xstream/XStreamExtensionProvider.java

```diff
@@ -99,6 +100,27 @@ public XStreamExtensionProvider(String namespace, String elementName, Class<?>..
 
     xstream.registerConverter(BooleanConverter.BINARY);
     xstream.registerConverter(new UrlEncodingStringConverter());
+
+    /*
+     * set map conversion format to
+     * <map><entry key="#key>" value="#value"/></map>
+     * instead of
+     * <map><entry><string>#key</string><string>#value</string></entry></map>
+     *
+     * breaks parsing compatibility with older clients if changed
+     */
+    xstream.registerConverter(
```

Comment:
If you really want to save byte you could use:
```
new NamedMapConverter(xstream.getMapper(), "entry", "key", String.class, null, Integer.class, true, false, xstream.getConverterLookup());

 <map>
   <entry key="keyValue">0</entry>
 </map>
 ```
 
But what if we are using maps of type string-object or integer-object where object is of type string, number, or boolean ?